src/main/java/com/thealgorithms/maths/FibonacciNumber.java

```diff
@@ -0,0 +1,24 @@
+import java.math.BigInteger;
+
+public final class FibonacciNumber {
+    private FibonacciNumber() {
+    }
+
+    // Compute the limit for n that fits in a long
+    private static final int argLimit = 92;
```

Comment:
I would suggest 70. And please change the description to something like _it gives wrong result for bigger inputs_.